hbw/app/assets/javascripts/hbw/components/form/select.js.jsx.coffee

```diff
@@ -80,7 +80,7 @@ modulejs.define 'HBWFormSelect',
 
       getChosenValue: ->
         if @props.params.mode == 'select'
-          if @props.value is null
+          if @props.value is null or @props.value is undefined
```

Comment:
This check looks suspicious. Looks like ```undefined``` is passed here when there is no BP variable, isn't it?
This line https://github.com/latera/homs/blob/master/hbw/app/assets/javascripts/hbw/components/form/group.js.jsx.coffee#L38

Check it, please.